core/common/src/SkyBox.ts

```diff
@@ -91,7 +91,7 @@ export type SkyBoxImageProps = SkySphereImageProps | SkyCubeImageProps | { type?
  *
  * The color and exponent properties are unused if one or more texture images are supplied.
  *
- * The color, exponent and texture properties are unused if an atmosphere is defined on the Environment.
+ * The skybox is ignored entirely when an atmosphere is defined on the Environment.
```

Comment:
I am reluctant to continue to belabor this but: the skybox is ignored when **display** of the atmosphere is enabled. An atmosphere (or skybox) can be defined but not enabled.

Reply:
Makes sense, fixed